python/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_packages
+
+setup(
+  name='dex',
+  version='0.0.1',
+  description='A research language for typed, functional array processing',
+  license='BSD',
+  author='Adam Paszke',
+  author_email='apaszke@google.com',
+  packages=find_packages(),
+  package_data={'dex': ['libDex.so']},
```

Comment:
BTW maybe we should check that this file exists either here at the top level, or by overriding some commands (e.g. the `build` command). We could give people a helpful error message that says they should run `make build-python` if it's not here. But it's fine to leave this for a follow-up.

Reply:
Good suggestion! I added a check at the top of the file.